save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationToolsMenu.kt

```diff
@@ -57,7 +57,40 @@ external interface OrganizationToolsMenuProps : Props {
     /**
      * lambda for update projects
      */
-    var updateProjects: (MutableList<ProjectDto>) -> Unit
+    var updateProjects: (List<ProjectDto>) -> Unit
+}
+
+/**
+ * This function delete [projects] by [projects], add [oldProject] in [projects] and sorted its by comparator
```

Comment:
~This function~ &mdash; redundant.

```suggestion
 * Removes the projects specified by [oldProjects], adds projects specified by [newProjects], and sorts the resulting list by their status and then by name.
```